### PR TITLE
chore: add tailwind css

### PR DESCRIPTION
**Pull request by Nigothazine**

### CODE REVIEW

File: .storybook/preview.ts [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
         color: /(background|color)$/i,
         date: /Date$/,
       },
     },
   },
+
+  decorators: [
+    // Adds theme switching support.
+    // NOTE: requires setting "darkMode" to "class" in your tailwind config
+    withThemeByClassName({
```
Comment (aceres) [inline .storybook/preview.ts:25]: It seems like duplicated withThemeByClassName here? Can we remove one of them?

File: .storybook/main.ts [hunk 2]
```diff
@@ -10,6 +10,14 @@ const config: StorybookConfig = {
     "@storybook/addon-essentials",
     "@storybook/addon-onboarding",
     "@storybook/addon-interactions",
+    {
```
Comment (aceres) [inline .storybook/main.ts:13]: Cool! 🤙

**Comment by aceres** [comment]
It works well on my side. Thanks for adding this in our config.

**Comment by aceres** [comment]
Be aware that we better should use Squash and Merge while merging this pull request 😎

File: .storybook/preview.ts (hunk 1 above)
Comment (Nigothazine) [reply .storybook/preview.ts:25]: yes, duplicate removed

File: .storybook/main.ts (hunk 2 above)
Comment (Nigothazine) [reply .storybook/main.ts:13]: 🥳

**Pull request merged**